http-server/src/main/java/io/airlift/http/server/HashLoginServiceProvider.java

```diff
@@ -42,18 +37,9 @@ public HashLoginServiceProvider(HttpServerConfig config)
     public HashLoginService get()
```

Comment:
`loadUsers()` method was throwing previously and it's removed in the new release, so nothing is throwing an `IOException` now. BTW that method was already a noop in 9.3.13.M0 so no ideas why we were calling it in the first place.